pegasus/src/homepage.rb

```diff
@@ -159,7 +159,7 @@ def self.get_blocks(request)
           color1: "0, 148, 202",
           color2: "89, 185, 220",
           url: "/educate",
-          image: "/shared/images/courses/logo_tall_teacher2.jpg",
+          image: "/images/homepage/ap-feature-2017.jpg",
```

Comment:
It's one of the files that changed in the `Files changed` section. There's just no preview because it's a binary file, according to github. You can preview it by clicking View.

Reply:
Oops, sorry to miss that.  I thought github used to preview images.

Reply:
That said, this image is currently very large.  We should use a resize URL to make it smaller dimensions when retrieving it.

Reply:
Oh, of course! I used the `.jpg` extension but forgot `fit-`. Fixing...